.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
Yeah, I just support it in too many places and removing that support is a major breaking change for all my projects - projects like Typings.